lib/user.js

```diff
@@ -10,3 +10,12 @@ export const isMuted = async ({ models, muterId, mutedId }) => {
 
   return !!mute
 }
+
+export const getUserById = async ({ models, userId, fields }) => {
+  const user = await models.user.findUnique({
+    where: { id: Number(userId) },
+    select: fields
+  })
+
+  return user
+}
```

Comment:
This helper function makes your code inconsistent with the rest of the codebase since we don't wrap db calls into their own functions. We just run them where they are needed.